### PR TITLE
Update ngOptions.js

### PR DESCRIPTION
**Pull request by epplestun**
Check value type to avoid errors

### CODE REVIEW

**Comment by epplestun** [comment]
I signed it!

File: src/ng/directive/ngOptions.js [hunk 1]
```diff
@@ -503,7 +503,7 @@ var ngOptionsDirective = ['$compile', '$parse', function($compile, $parse) {
             option.element.selected = false;
           });
 
-          if (value) {
+          if (value && Array.isArray(value)) {
```
Comment (gkalpak) [inline src/ng/directive/ngOptions.js]: It's better to use Angular's `isArray()`.

**Comment by epplestun** [comment]
Why is better?

**Comment by gkalpak** [comment]
It is better if we use the same function for detecting arrays throughout the codebase, becaue that way it is easier to fix issues or add features.
E.g. right now, `isArray()` is just a wrapper for `Array.isArray()`, but it was not always the case (e.g. on older versions supporting IE8) and it might not alaways be the case.

By using the same function, we only have to make the change in one place and we get more consistent behaviour across different parts (e.g. imagine one directive recognizing something as array, while another won't).

**Comment by epplestun** [comment]
Ok, I will change Array.isArray(value) by angular.isArray(value) :)

**Comment by gkalpak** [comment]
BTW, you didn't have to close this one. If you pushed a new commit to your branch or amended the previous commit (and force-pushed), GitHub would automatically update the PR.

**Pull request closed without merging**